index.html

```diff
@@ -181,7 +182,7 @@ <h4>LET'S CONNECT</h4>
         <ul>
           <li><input type="text" maxlength="30" placeholder="Name" required></li>
           <li><input type="email" placeholder="Enter E-mail" required></li>
```

Comment:
- [ ] (OPTIONAL) Kindly apply the same placeholders as shown in the design. That is `Yeremias` and `natajanyanji14@gmail.com`. This will make your site to be more consistent with the Figma design 👍